main/settings.py

```diff
@@ -1537,3 +1537,9 @@
     default=None,
     description="Password for Trino authentication",
 )
+
+VC_SIGNER_URL = get_string(
```

Comment:
consider adding VC_SIGNER_URL=http://dcc.odl.local:4005/instance/test/credentials/issue to the [example env file](https://github.com/mitodl/mitxonline/blob/main/.env.example)